mypython.py

```diff
@@ -0,0 +1,4 @@
+first_name=('tanisha')
+second_name=('ashika')
+names=first_name+ '' +second_name
```

Comment:
Please remove the single quotes in the line 3 and let me know

Reply:
i removed single quote.pklease check

Reply:
Good.. I will approve now